pegasus/src/homepage.rb

```diff
@@ -431,17 +416,4 @@ def self.get_below_hero_announcement
       link_text: "homepage_below_hero_announcement_link_text"
     }
   end
-
-  def self.get_dance_stars
```

Comment:
This is also used on /dance, https://github.com/code-dot-org/code-dot-org/blob/88febc6b7a4bfe9bdc8a53488ded04c255caeb1e/pegasus/sites.v3/code.org/views/dance_tutorial_section.haml#L35 so I don't think we want to delete it yet.